src/CirclicalUser/Mapper/UserResetTokenMapper.php

```diff
@@ -65,11 +65,12 @@ public function get(int $tokenId)
     public function invalidateUnusedTokens(AuthenticationRecordInterface $authenticationRecord)
     {
         $query = $this->getRepository()->createQueryBuilder('r')
-            ->update()
+            ->update('')
             ->set('r.status', UserResetTokenInterface::STATUS_INVALID)
             ->where('r.authentication = :authentication')
-            ->andWhere('r.status', UserResetTokenInterface::STATUS_UNUSED)
+            ->andWhere('r.status = :status_unused')
             ->setParameter('authentication', $authenticationRecord)
```

Comment:
Please combine the setParameter calls into a single setParameters call that accepts an array.
http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/query-builder.html